### PR TITLE
Update American customer bank info

### PR DESCRIPTION
**Pull request by Lauren-penner**
@jnguyen1990 
Add bank_account_type and authorization_type to American customer and add notice in transaction creation about customer requirements.

### CODE REVIEW

File: source/includes/_transaction_schedules.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 Transaction schedules are the method by which recurring or one time payments are scheduled in Rotessa for a customer. Payments require a schedule date, which must be at least 2 business days in the future, as well as a frequency.
 
+<aside class="notice">
+A customer must have bank information complete in order to make a transaction schedule for them. 
+<br/>
+Canadian customers require <code>institution_number</code>, <code>transit_number</code>, and <code>account_number</code>.
+<br/>
+American customers require <code>routing_number</code>, <code>bank_account_type</code>, <code>authorization_type</code>, and <code>account_number</code>.
```
Comment (jnguyen1990) [inline source/includes/_transaction_schedules.md]: @Lauren-penner 
Put routing number next to account number please

File: source/includes/_transaction_schedules.md [hunk 2]
```diff
@@ -107,6 +115,14 @@ comment | Optional comment for schedule
 When creating schedules you must specify a process date of at least 2 business days in the future.
 </aside>
 
+<aside class="notice">
+A customer must have bank information complete in order to make a transaction schedule for them. 
+<br/>
+Canadian customers require <code>institution_number</code>, <code>transit_number</code>, and <code>account_number</code>.
+<br/>
+American customers require <code>routing_number</code>, <code>bank_account_type</code>, <code>authorization_type</code>, and <code>account_number</code>.
```
Comment (jnguyen1990) [inline source/includes/_transaction_schedules.md]: Here too

File: source/includes/_transaction_schedules.md [hunk 3]
```diff
@@ -151,6 +167,14 @@ comment | Optional comment for schedule
...
```
Comment (jnguyen1990) [inline source/includes/_transaction_schedules.md]: ditto

Comment (Lauren-penner) [reply source/includes/_transaction_schedules.md]: 👍

File: source/includes/_transaction_schedules.md (hunk 2 above)
Comment (Lauren-penner) [reply source/includes/_transaction_schedules.md]: 👍

File: source/includes/_transaction_schedules.md (hunk 1 above)
Comment (Lauren-penner) [reply source/includes/_transaction_schedules.md]: 👍

**Comment by jnguyen1990** [comment]
👍

**Pull request merged**